pkg/setup.cfg

```diff
@@ -67,7 +67,7 @@ release = ___release___
 build_requires = python >= 2.6
 requires = python >= 2.6
            cmsgemos_gempython >= 0.3.1
-           cx_Oracle
+           cx_Oracle <= 5.2.1
```

Comment:
I don't believe this is a strict requirement, however, it **is** a requirement that couples with the version of `sqlalchemy` that is installed (if `sqlalchemy` is too old, then it requires an older version of `cx_Oracle`)

Reply:
So you're suggesting the requirement on version number should move to `sqlalchemy`?

Reply:
No, rather I'm saying that I wouldn't implement this change here at all, because the problem *isn't* with this package... (the brute force ~~solution~~ hack would be to restrict *both* to exactly the versions being used...)

Reply:
What would be your preferred solution then.  From what I understood the problem was an upstream change in `cx_Oracle` and `sqlalchemy` that made an issue for `pandas` `DataFrame:: read_sql`.  The change here resolves this problem and restricts the packages to what is natively available in `cc7` which I understood was one of your guiding stars, e.g. do not introduce additional dependencies required on GEM machines which would necessitate new RPM's and dependencies to be built and packaged for `puppet`.

Reply:
> What would be your preferred solution then?
> > No, rather I'm saying that I wouldn't implement this change here at all

I think I already stated my preference...
Yes, your change here does ~~"fix"~~ mitigate the issue for `cc7`, but in 3-4 months, we'll have a `cc8` build for testing, so my earlier preference to test against different versions (`sqlalchemy` and `cx_Oracle`) to find a robust solution that would work regardless, is my higher preference.  At present, it's trivial to ensure that the `cx_Oracle` version is pegged on the machines being used.
In fact, I just checked and on most machines, the `cx_Oracle` version is `5.2.1-1`, while on two (`gem904qc8dqm` and `gem904daq04`) the version is `7.1-2`

Reply:
> Yes, your change here does "fix" mitigate the issue for cc7, but in 3-4 months, we'll have a cc8 build for testing, so my earlier preference to test against different versions (sqlalchemy and cx_Oracle) to find a robust solution that would work regardless, is my higher preference. At present, it's trivial to ensure that the cx_Oracle version is pegged on the machines being used.

I would suggest that you open a separate issue to address this in the future and allow this  for the time being as we have limited resources and this mitigation allows work to continue without taking time away from higher priority tasks.

Reply:
How about we just change as per https://github.com/cms-gem-daq-project/gem-plotting-tools/issues/240#issuecomment-520614525 and bypass the need for an explicit versionlock altogether

Reply:
second this ^^

Reply:
> How about we just change as per [#240 (comment)](https://github.com/cms-gem-daq-project/gem-plotting-tools/issues/240#issuecomment-520614525) and bypass the need for an explicit versionlock altogether

I've implemented this solution and the offending commit was been removed.